### ISSUE TITLE
allow other size units than px

### ISSUE THREAD

**Issue by Freundschaft** [kadencethemes/kadence-blocks#22]
Its kind of annoying that the block configurator usually forces you to resort to px values, but sometimes i need to use values such as rem, em, or vh/vw

is there a reason why those are not selectable in the block configurator?

Thanks!

**Comment by kadencethemes** [comment]
Hey, thanks for posting can you tell me what specific controls? I can look at adding these options.

Ben

**Comment by Freundschaft** [comment]
it would be useful for every place where I can set a size, but for me I need it for the advanced heading setting, as shown in this screenshot:

![image](https://user-images.githubusercontent.com/5065616/52977124-60aff500-33cc-11e9-93a7-fe0a500c15f2.png)

**Comment by Freundschaft** [comment]
Also it would be especially useful for setting the height of the spacer block

**Comment by kadencethemes** [comment]
ok, I can look at adding rem to the font size, You can already set it to em and make that the default for the block settings.

**Comment by Freundschaft** [comment]
thanks! any reason why viewport units are not supported?

**Comment by kadencethemes** [comment]
just not seem them used for fonts much but certainly can be added to the list.

**Comment by Freundschaft** [comment]
yes sorry, I meant for the spacer block and margins

**Comment by kadencethemes** [comment]
Spacer will have in the next update, working on adding to margins as well.

**Comment by Freundschaft** [comment]
thanks a lot!

**Comment by Freundschaft** [comment]
also I noticed for the row layout block, the content max width and minimum height can only be set in pixel, which is kind of inconvenient

**Comment by kadencethemes** [comment]
Row layout min and max height now have multiple unit options margins as well.

**Comment by lydenyardley** [comment]
I size all my padding and margins in rem's and I see this option is missing everywhere I've looked so far.

The reason I do this is to **maintain consistent vertical rhythm**.
So I set my html element to `font-size: 100%`, producing 16px, and then increase incrementally at various breakpoints upto 20px. Then I have my line-height set to 1.6. This line-height becomes my baseline unit. So now, I set my text top & bottom margins to 1.6rem, same for box padding, and  other areas. Now the entire design scales with the base font size, which changes based on viewport width. Make sense?

Can this be added, or perhaps setup in master settings somewhere, so that you don't overwhelm users with every single measurement unit there is?
